src/rules/effects/prefer-concat-latest-from.ts

```diff
@@ -21,43 +30,96 @@ export default createRule<Options, MessageIds>({
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
     messages: {
-      [messageId]: 'Use `concatLatestFrom` instead of `withLatestFrom`.',
-      [messageIdSuggest]: 'Replace `withLatestFrom` with `concatLatestFrom`.',
+      [messageId]: `Use \`${concatLatestFromKeyword}\` instead of \`${withLatestFromKeyword}\`.`,
     },
   },
-  defaultOptions: [],
-  create: (context) => {
+  defaultOptions: [defaultOptions],
+  create: (context, [options]) => {
+    const sourceCode = context.getSourceCode()
+    const selector = getSelector(context, options)
+
+    if (!selector) {
+      return {}
+    }
+
     return {
-      [`${createEffectExpression} > [body] > [arguments] > CallExpression > Identifier[name='withLatestFrom']`](
-        node: TSESTree.Identifier,
-      ) {
+      [selector](node: WithLatestFromIdentifier) {
         context.report({
           node,
           messageId,
-          suggest: [
-            {
-              messageId: messageIdSuggest,
-              fix: (fixer) => {
-                return [fixer.replaceText(node, 'concatLatestFrom')].concat(
-                  getImportAddFix({
-                    fixer,
-                    importName: 'concatLatestFrom',
-                    moduleName: NGRX_MODULE_PATHS.effects,
-                    node,
-                  }),
-                )
-              },
-            },
-          ],
+          fix: (fixer) => getFixes(sourceCode, fixer, node),
         })
       },
     }
   },
 })
+
+function getSelector(
+  context: Readonly<TSESLint.RuleContext<MessageIds, Options>>,
+  { strict }: Options[number],
+) {
+  if (strict) {
+    return `${createEffectExpression} CallExpression > Identifier[name='withLatestFrom']` as const
+  }
+
+  const actionsName = findNgRxEffectActionsName(context)
+  return actionsName
+    ? (`${createEffectExpression} ${storeExpression(
+        actionsName,
+      )} > CallExpression[arguments.length=1] > Identifier[name='${withLatestFromKeyword}']` as const)
+    : null
+}
+
+function getFixes(
+  sourceCode: Readonly<TSESLint.SourceCode>,
+  fixer: TSESLint.RuleFixer,
+  node: WithLatestFromIdentifier,
+) {
+  const { parent } = node
+  const isUsingDeprecatedProjectorArgument = parent.arguments.length > 1
+  const [firstArgument] = parent.arguments
+  const nextToken =
+    isUsingDeprecatedProjectorArgument &&
+    sourceCode.getTokenAfter(firstArgument)
+  return [
+    fixer.replaceText(node, concatLatestFromKeyword),
+    ...(isArrowFunctionExpression(firstArgument)
+      ? []
+      : [fixer.insertTextBefore(firstArgument, '() => ')]),
+  ].concat(
+    getImportAddFix({
+      fixer,
+      importName: concatLatestFromKeyword,
+      moduleName: NGRX_MODULE_PATHS.effects,
+      node,
+    }),
+    ...(isUsingDeprecatedProjectorArgument && nextToken
+      ? [
+          getImportAddFix({
+            fixer,
+            importName: 'map',
+            moduleName: 'rxjs',
```

Comment:
This is the "problem" I was mentioning before... should we use `rxjs` or `rxjs/operators`?

Reply:
`rxjs/operators` 
We'll figure out how we want to use the rxjs imports later

Reply:
Reason: because that's what peoples are used to now, and we're sure that we don't break anything 😅